sqrl/sqrl.py

```diff
@@ -141,7 +141,7 @@ def sqrl_url(self):
         """
         return (
             '{scheme}://{host}{url}'
-            ''.format(scheme='sqrl' if self.request.is_secure() else 'qrl',
+            ''.format(scheme='sqrl' if self.request.is_secure() else 'sqrl',
```

Comment:
This is one change I need to revert. I'm not sure we should present a QR code at all in the instance that SSL isn't here. I had some trouble trying to dev this locally without an SSL cert, so I ended up slapping one on the dev environment. The client I was testing with assumed TLS, so it wouldn't work even with the correct prefix.